src/cipher/cipher.rs

```diff
@@ -251,45 +242,3 @@ impl PoseidonCipher {
         [domain, length, ks0, ks1, nonce]
     }
 }
-
-impl io::Write for PoseidonCipher {
```

Comment:
I thought it was used for the from / to bytes impl, but good to know we can safely remove it without changing the implementation of the methods above!

Reply:
Yep, it was meant to be an extension and never a requirement